example/flutter/flutter_example.dart

```diff
@@ -0,0 +1,64 @@
+// Copyright 2016, the Flutter project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:usage/usage.dart';
+
+import 'package:flutter/material.dart';
+
+import 'dart:io';
+
+void main() {
```

Comment:
We may want to re-do this in markdown, as a few lines diffs to the std flutter hello-world app.